cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -103,6 +105,14 @@ object EvaluatorCommand : CommandWithHelp() {
     )
     private var packageCurationsFile: File? = null
 
+    @Parameter(
+        description = "A YAML that contains the license configuration. That license configuration is passed as " +
```

Comment:
"A YAML **file**...". Technically this could also be a JSON or XML file, because you use `OutputFormat.readValue()`.

Reply:
The writing was a bit inspired by above descriptions, see e.g. `ort.yml` or `YAML file`.
I changed it anyway.